tests/kani/Intrinsics/PtrOffsetFromUnsigned/check_unsigned_ptr_offset_from.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
```

Comment:
Please add a small description for the test here: "Checks that the `ptr_offset_from_unsigned` intrinsic returns the expected results." or similar.